.gitignore

```diff
@@ -8,3 +8,6 @@ buildfiles
 /.gradle/
 /reports/
 /generated/
+*.iml
```

Comment:
Do we really need .iml and .ipr? They seem to be legacy metadata formats. https://intellij-support.jetbrains.com/hc/en-us/articles/206544839